### PR TITLE
Update VLC version

### PR DESCRIPTION
**Pull request by TheTedder**
This series of commits updates the component to use the VLC nuget package instead of importing directly from the binaries. This not only means the latest VLC version is supported but it removes several dependencies and significantly cleans up the code.
There is one known issue: sometimes videos will start at the beginning even when an offset is specified.

### CODE REVIEW

**Comment by TheTedder** [comment]
@wooferzfg I would really like to get this merged before livesplit 1.8 comes out so if you could look this over and make sure it's satisfactory I would be very happy. This PR addresses #5 btw.

File: VideoComponent.cs [hunk 1]
```diff
@@ -28,64 +32,87 @@ private class VLCErrorException : Exception { }
...
+                Control.Invoke((InvokeAction)Lock, Control, x);
+            else
+                Lock(Control,x);
+        }
+
         static void ErrorCallback(Form form, Exception ex)
         {
-            string requiredBits = Environment.Is64BitProcess ? "64" : "32";
-            MessageBox.Show(form, "VLC Media Player 2.2.1 (" + requiredBits + "-bit) along with the ActiveX Plugin need to be installed for the Video Component to work.", "Video Component Could Not Be Loaded", MessageBoxButtons.OK, MessageBoxIcon.Error);
+            //string requiredBits = Environment.Is64BitProcess ? "64" : "32";
```
Comment (wooferzfg) [inline VideoComponent.cs:88]: Could you get rid of all the commented out code introduced in this PR, such as this?

File: LiveSplit.Video.sln [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29123.88
```
Comment (wooferzfg) [inline LiveSplit.Video.sln:4]: I don't think this change is required?

File: LiveSplit.Video.csproj [hunk 3]
```diff
@@ -83,26 +95,27 @@
...
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+    </PropertyGroup>
+    <Error Condition="!Exists('packages\VideoLAN.LibVLC.Windows.3.0.7.1\build\VideoLAN.LibVLC.Windows.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\VideoLAN.LibVLC.Windows.3.0.7.1\build\VideoLAN.LibVLC.Windows.targets'))" />
+    <Error Condition="!Exists('packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets'))" />
+  </Target>
+  <Import Project="packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets" Condition="Exists('packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" />
+  <PropertyGroup>
+    <PostBuildEvent>xcopy $(TargetDir)LiveSplit.Video.dll E:\dambu\Documents\LiveSplit_1.7.6\Components /Y
+xcopy $(TargetDir)LiveSplit.Video.pdb E:\dambu\Documents\LiveSplit_1.7.6\Components /Y</PostBuildEvent>
```
Comment (wooferzfg) [inline LiveSplit.Video.csproj:117]: These are your local paths and won't work for other people. This occurs in other places in this PR as well.

**Review by wooferzfg — CHANGES_REQUESTED** [review]
I'm hesitant to merge this until the known issue that you mention gets addressed. Therw will be minor releases after LiveSplit 1.8, so I don't think we need to rush this. Also, we'll need to make sure that this works with the CI build in the main LiveSplit repo.

File: VideoComponent.cs (hunk 1 above)
Comment (TheTedder) [reply VideoComponent.cs:88]: Yes. Also I just realized this error message no longer applies since we're not relying on ActiveX anymore.

**Comment by wooferzfg** [comment]
New WIP PR is here: https://github.com/LiveSplit/LiveSplit.Video/pull/7

**Pull request closed without merging**